index.html

```diff
@@ -97,6 +97,16 @@
       </div>
     </header>
 
+    <div class="alert" id="community-map-alert">
+      <span
+        class="alert-close-button"
+        onclick="this.parentElement.style.display='none';"
+        >&times;</span
+      >
```

Comment:
I posted in Slack confirming we want to allow people to close it. Please don't merge this PR until that's confirmed

Reply:
Imo let's remove this closing mechanism for now. We can always add it back if people complain.